code/modules/mapping/map_config.dm

```diff
@@ -1,143 +1,178 @@
-//used for holding information about unique properties of maps

```

Comment:
Whitespace changes in this file could be fixed? It's a bit difficult to diff changes with this -143,+178 (also happened to me previously)

Reply:
warning: LF will be replaced by CRLF in code/modules/mapping/map_config.dm.
The file will have its original line endings in your working directory

Reply:
do https://github.com/tgstation/tgstation/pull/51412/files?w=1 for now i suppose
git literally will not let me change it back because it's autoconverting 
any suggestions folks?

Reply:
Unfuck our `.gitattributes` tbqh fam